cranelift/codegen/src/machinst/isle.rs

```diff
@@ -804,6 +804,18 @@ macro_rules! isle_prelude_methods {
             self.lower_ctx.use_constant(data)
         }
 
+        #[inline]
+        fn emit_u128_le_const(&mut self, lower: u64, upper: u64) -> VCodeConstant {
```

Comment:
We can probably use a `u128` here to avoid the awkward iter-chaining below? Then we can do a zero-extend cast when calling this rather than giving `0` as `upper`.